app/src/main/java/org/intelehealth/app/activities/familyHistoryActivity/FamilyHistoryActivity.java

```diff
@@ -206,7 +207,7 @@ public void onClick(DialogInterface dialog, int which) {
                     intent.putExtra("EncounterAdultInitial_LatestVisit", EncounterAdultInitial_LatestVisit);
                     intent.putExtra("state", state);
                     intent.putExtra("name", patientName);
-                    intent.putExtra("patientFirstName",patientFName);
+                    intent.putExtra("patientFirstName", patientFName);
```

Comment:
Use constant variable instead of direct use of static strings as params to putExtra
`intent.putExtra({require constant variable }, patientFName);`